docs/source/concepts/workarounds.rst

```diff
@@ -74,7 +74,7 @@ The ``pipeline.yaml`` section for the cloneable section diagrammed above might l
    :emphasize-lines: 7,12,17,22
 
    determining_exclusions_and_removing_records:
-     parallel:
+     clones:
```

Comment:
This is the primary purpose of this PR as it's user-facing. @zmbc did not provide an alternative to "parallel" in the ticket description, so I just went with what makes the most sense to me "clones".